androidapp/DansFrappesRaps/app/src/main/java/com/cs3450/dansfrappesraps/ui/screens/EditMenuScreen.kt

```diff
@@ -0,0 +1,93 @@
+package com.cs3450.dansfrappesraps.ui.screens
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
```

Comment:
> I would also maybe change the delete swipe action to include the entire card. Right now it is fixed I think to the size of the garbage logo.

Intentionally leaving it this way for now so that there can't be a misclick to delete, and when it is stretched to match the card size it can look a little weird. We can look at it again later when reviewing the UI for the whole app